cpu/nrf52/include/periph_cpu.h

```diff
@@ -161,12 +163,13 @@ typedef struct {
     gpio_t pin[PWM_CHANNELS];           /**< PWM out pins */
 } pwm_conf_t;
 
-#ifdef CPU_MODEL_NRF52840XXAA
+#if defined(CPU_MODEL_NRF52811XXAA) || defined(CPU_MODEL_NRF52840XXAA)
 /**
  * @brief   Structure for UART configuration data
  */
 typedef struct {
-    NRF_UARTE_Type *dev;    /**< UART with EasyDMA device base register address */
+    NRF_UARTE_Type *dev;    /**< UART with EasyDMA device base
+                             * register address */
```

Comment:
No need for that line break.

Reply:
Otherwise the line is too long. It was a warning in Travis.